app/views/application/personal-information/residency.njk

```diff
@@ -12,11 +12,14 @@
 {% block primary %}
 
 <p class="govuk-body">To get help with student visas and your immigration status, speak to a <a href="https://beta-adviser-getintoteaching.education.gov.uk/">Get Into Teaching</a> adviser.</p>
+<h2 class="govuk-heading-m">Getting a visa through your training provider</h2>
+<p class="govuk-body">You may be able to get a visa through your teacher training provider. This is called sponsoring a student visa.</p>
+<p class="govuk-body">It depends on what type of training provider they are and if they have a student visa sponsor licence.</p>
 <p class="govuk-body">Universities providing initial teacher training can usually sponsor student visas for courses without a salary.</p>
-<p class="govuk-body">State funded schools, including those that are School Centred Initial Teacher Training (SCITT) providers, cannot sponsor student visas for courses without a salary.</p>
+<p class="govuk-body">State funded schools, including School Centred Initial Teacher Training (SCITT) providers, cannot sponsor student visas for courses without a salary.</p>
+<p class="govuk-body">Contact your chosen training provider if you're not sure if they are university or a SCITT.</p>
 <p class="govuk-body"><a href="https://www.gov.uk/government/publications/register-of-licensed-sponsors-students">Check if a provider has a student visa sponsor licence</a>.</p>
-<p class="govuk-body">You may be eligible for another type of visa that does not need your training provider to act as sponsor.</p>
-<p class="govuk-body"><a href="https://beta-getintoteaching.education.gov.uk/international-candidates">Find out about student visas, visa routes that do not need a sponsor and visa options for trainees on courses with a salary</a>.</p>
+<p class="govuk-body">Find out more about your visa options on <a href="https://beta-getintoteaching.education.gov.uk/international-candidates">Get Into Teaching</a>.</p>
```

Comment:
Where we deep-link into Get Into Teaching, I think it’s a good idea to include some reference to the page we’re linking to within the link text, otherwise it can be a bit disorientating. (Better for screen readers too).

I’m also not sure that page is hugely helpful here, as it’s not _just_ about visas, and the visa section mostly just contains a link to GOV.UK anyway...?